src/charm.py

```diff
@@ -42,11 +44,29 @@ def __init__(self, *args):
         self.framework.observe(self.on.update_status, self._on_update_status)
         self._provide_mysql()
 
+    def _mysql_layer(self):
+        """Construct the pebble layer"""
+        logger.debug("Building pebble layer")
+        layer = {
```

Comment:
Minor nit, but could just replace this with:

```
return {
...
}
```

Reply:
Done